tests/test-helper.js

```diff
@@ -5,7 +5,7 @@ setResolver(resolver);
 
 /* jshint ignore:start */
 mocha.setup({
-    timeout: 15000,
+    timeout: 25000,
```

Comment:
Only the editor tests, perhaps we can do it within the describe rather than the it?